packages/account/src/Sections/Verification/ProofOfIdentity/onfido-sdk-view-container.tsx

```diff
@@ -42,6 +43,7 @@ const OnfidoSdkViewContainer = ({
     getChangeableFields,
     handleViewComplete,
     height,
+    is_disabled = true,
```

Comment:
Why we need this default value to be true?

Reply:
adding a new prop to handle specific scenario for NIgeria, mean we only need onfido to be enabled in that case so adding default to true else we would need to pass from everywhere it's been used

Reply:
what if we do this: 
`
....,
height,
is_disabled,`

`React.useEffect( () => {
    if(is_disabled) {
        setIsOnfidoDisabled(false)
    }
}, [is_disabled])`

then from the component

`<OnfidoUpload
  ......
  is_disabled
  .......
/>`

since `is_disabled` is `boolean | undefined` I think it's fine to not pass from other component which doesn't include `is_disabled`

Reply:
@suisin-deriv updated. Thanks for your suggestion 👍